packages/react/src/themes/teams-high-contrast/components/Menu/menuVariables.ts

```diff
@@ -1,13 +1,26 @@
+import { pxToRem } from '../../../../lib'
+
 export interface MenuVariables {
   color: string
   activeColor: string
   activeBackgroundColor: string
+  selectedBackgroundColor: string
```

Comment:
what's the difference between `selectedBackgroundColor` and `activeBackgroundColor` we should always aim for meaningful names

Reply:
selectedBackgroundColor: selected/preselected item in the menu
activeBackgroundColor: hover & focus state of the menuitem

Happy to change the names to something else if you've a recommendation.

Reply:
got it

> selectedBackgroundColor: selected/preselected item in the menu

We're using the `active` prop to describe selected items in the menu, so a better name would actually be `activeBackgroundColor`

Now that name is used for hover & focus state already and it was there before this PR was introduced so it was an error on our side.

Therefore, I'd recommend changing:
`activeBackgroundColor` -> `focusedBackgroundColor` (which already exists, can we reuse it) if not we need to understand why
`selectedBackgroundColor` -> `activeBackgroundColor`

What do you think @jaanus03 ?

Reply:
Thanks @Bugaa92, this makes perfect sense. I've updated the names.